### PR TITLE
Fix connection error in case of audio/video call in Google Meet

### PR DESCRIPTION
**Pull request by mahadevans87**
Google meet is unusable - Errors out when trying to connect to a new audio/video call. Google Hangouts had a similar issue and the same user-agent fix worked.

 - Strip off any Ferdi/Electron text from user-agent using modifyRequestHeaders hook
 - Resize icon to 1024x1024px as npm package was complaining
 - Bump version number to 1.0.3

**Screenshots**

Before
meet_error

After
google_meet_fix

### CODE REVIEW

**Comment by sampathBlam** [comment]
@kytwb or @vantezzen  : Can you please repackage this ? We are still facing the travis CI issue when packaging.

File: uncompressed/googlemeet/index.js [hunk 1]
```diff
@@ -2,10 +2,16 @@ var os = require('os')
 
 module.exports = Franz =>
   class Gmail extends Franz {
```
Comment (sampathBlam) [inline uncompressed/googlemeet/index.js:4]: @mahadevans87 : Can we change the class name to something like GoogleMeet? Probably, the recipe was copied from Gmail.

Comment (mahadevans87) [reply uncompressed/googlemeet/index.js:4]: Yeah makes sense

Comment (mahadevans87) [reply uncompressed/googlemeet/index.js]: Done

**Comment by sampathBlam** [comment]
@mahadevans87 : Can we remove the changes from the commit https://github.com/getferdi/recipes/pull/186/commits/d9ca0cbebd85f11ef2517411559ceb00895400d0 in this PR ? Seems like it got in by mistake.

**Pull request merged**